1_fetch/src/get_nwis_data.R

```diff
@@ -5,9 +5,9 @@ generate_files <- function(site_nums = c("01427207", "01432160", "01435000", "01
   
   return(download_files)
 }
+# 2014-05-01
 
-
-download_nwis_site_data <- function(filepath, parameterCd = '00010', startDate="2014-05-01", endDate="2015-05-01"){
+download_nwis_site_data <- function(filepath, parameterCd = '00010', startDate="2013-05-01", endDate="2015-05-01"){
```

Comment:
Did this change trigger a rebuild in the original code?

Reply:
No - not until I edited the yaml file, as you suggested

Reply:
Great, was it revealing to see the difference? Does it make sense why something similar to the second approach was needed?

Reply:
Yes - that makes complete sense.

Reply:
Sorry - on further reflection, I 100% get _why_ such an approach was needed, but I'm less clear on _how_ that approach solved the problem. I'll ask my question in the chat

Reply:
Gotcha. Do you feel solid about it after the exchange in the chat?

Reply:
Yes - Alison's explanation made a lot of sense. I had been confused by the printed output, which traces the tree _forwards_. I didn't realize that scipiper was first invisibly tracing the tree _backwards_ in order to determine what had to be updated, and then tracing the tree back _forwards_ as it made those updates. But now it all makes sense